src/include/duckdb/common/file_system.hpp

```diff
@@ -187,6 +190,9 @@ class FileSystem {
 	//! Extract the name of a file (e.g if the input is lib/example.dll the name is 'example.dll')
 	DUCKDB_API static string ExtractName(const string &path);
 
+	//! Returns the value of an environment variable - or the empty string if it is not set
+	DUCKDB_API static string GetEnvVariable(const string &name);
```

Comment:
This should probably not be filesystem specific, and be used here too: https://github.com/duckdb/duckdb/blob/e466d90508a4422179068bb3a94c0c44f239155e/extension/httpfs/s3fs.cpp#L145

Reply:
We should clean this up in the future - but currently the `FileSystem` class holds almost all OS-specific code. In this case as a static method it is not file-system specific but only OS-specific.

I will leave this as-is for now in light of trying to get this finished before the release, but that definitely should also be updated later on.